atomic_reactor/build.py

```diff
@@ -157,6 +157,8 @@ def __init__(self, source, image, **kwargs):
         self.built_image_info = None
         self.image = ImageName.parse(image)
         self.base_from_scratch = False
+        self.custom_base_image = False
+        self.custom_parent_image = False
```

Comment:
Would be great to have comments explaining what these mean -- it isn't immediately obvious to me from the names.

Reply:
yes I'll add comments